ironic/drivers/modules/drac/raid.py

```diff
@@ -293,6 +293,43 @@ def clear_foreign_config(node, raid_controller):
                    'error': exc})
         raise exception.DracOperationError(error=exc)
 
+def change_physical_disk_state(node, mode,
+                               controllers_to_physical_disk_ids=None):
+    """
+    :param node: an ironic node objec
```

Comment:
Can you change "objec" to "object" in the line above?

Reply:
Done